Polyclinic/Polyclinic.Infrastructure.PostgreSQL/Repository/PostgresAppointmentRepository.cs

```diff
@@ -0,0 +1,134 @@
+using Microsoft.EntityFrameworkCore;
+using Polyclinic.Domain.Interfaces;
+using Polyclinic.Domain.Subjects;
+
+namespace Polyclinic.Infrastructure.PostgreSQL.Repository;
+
+public class PostgresAppointmentRepository(PolyclinicDbContext context) : IRepository<Appointment, int>
+{
+    /// <summary>
+    /// Create new appointment
+    /// </summary>
+    public int Create(Appointment entity)
+    {
+        context.Appointments.Add(entity);
+        context.SaveChanges();
+        return entity.Id;
+    }
+
+    /// <summary>
+    /// Read all appointments
+    /// </summary>
+    public List<Appointment> ReadAll()
+    {
+        return context.Appointments
+            .Include(a => a.Patient)
+            .Include(a => a.Doctor)
+                .ThenInclude(d => d!.Specialization)
+            .ToList();
```

Comment:
Упростить инициализацию.
```suggestion
        return [.. context.Appointments
            .Include(a => a.Patient)
            .Include(a => a.Doctor)
            .ThenInclude(d => d.Specialization)];
```